mne_connectivity/tests/test_spectral.py

```diff
@@ -91,14 +91,21 @@ def test_spectral_connectivity(method, mode):
         else:
             mt_bandwidth = None
 
-        con, freqs, times, n, _ = spectral_connectivity(
+        con, _ = spectral_connectivity(
             data, method=method, mode=mode, indices=None, sfreq=sfreq,
             mt_adaptive=adaptive, mt_low_bias=True,
             mt_bandwidth=mt_bandwidth, cwt_freqs=cwt_freqs,
             cwt_n_cycles=cwt_n_cycles)
 
+        if isinstance(method, list):
+            freqs = con[0].freqs
+            n = con[0].n_epochs
+        else:
+            freqs = con.freqs
+            n = con.n_epochs
+
         assert (n == n_epochs)
-        assert_array_almost_equal(times_data, times)
+        # assert_array_almost_equal(times_data, times)
```

Comment:
I think it's probably still a good idea to test that the time vector doesn't get mangled in the connectivity function.

Reply:
So it looks like the `times_data` vector is just the tmin:tmax inside `_prepare_connectivity()`:

```
    mask = _time_mask(times_in, tmin, tmax, sfreq=sfreq)
    tmin_idx, tmax_idx = np.where(mask)[0][[0, -1]]
    tmax_idx += 1
    tmin_true = times_in[tmin_idx]
    tmax_true = times_in[tmax_idx - 1]  # time of last point used

    times = times_in[tmin_idx:tmax_idx]
```

Is it necessary to keep that?